src/replacements/tns/index.ts

```diff
@@ -1,7 +1,7 @@
 import type { FindAndReplace } from '../index.js'
 
 export default {
-  find: /AT\s*(2\d{3}[a-z]+)/g,
+  find: /(?<!http:\/\/|https:\/\/)(?<!http:\/\/.*|https:\/\/.*)AT\s*(2\d{3}[a-z]+)/g,
```

Comment:
You may be able to simplify the regex:
```suggestion
  find: /(?<!http(s?):\/\/)(?<!http(s?):\/\/.*)/g,
```
the `s?` should match 0 or 1 of the character `s`, can you try this and see if it works?

Reply:
that seems to work, thanks!